src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.File.Move.cs

```diff
@@ -50,6 +50,7 @@ public void Linux_File_Move_To_Different_Watched_Directory()
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.OSX)] // Not supported since OSX 10.15 Issue: https://github.com/dotnet/corefx/issues/40034
```

Comment:
Can you use `[ActiveIssue(40034, TestPlatforms.OSX)]` instead?

Reply:
Sure